docs/metal_vrf.md

```diff
@@ -31,6 +31,10 @@ VRF operations : It defines a collection of customer-managed IP blocks that can
 
 * [metal](metal.md)	 - Command line interface for Equinix Metal
 * [metal vrf create](metal_vrf_create.md)	 - Creates a Virtual Routing and Forwarding(VRF) for a specified project.
+* [metal vrf createRoute](metal_vrf_createRoute.md)	 - Create a route in a VRF. Currently only static default routes are supported.
```

Comment:
The linked issue suggests `routes-create`, `routes-update`, etc., which is more in line with how we generally format subcommand names.

Another option that I personally prefer would be to promote route operations to a separate subcommand, rather than adding them in the `vrf` subcommand.  That could look like either `metal vrf route <verb>` (e.g. `metal vrf route create --vrf=<vrf_id>`) or `metal vrf-route <verb>` (`metal vrf-route create --vrf-id`).